tests/ui/test.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import os
+
+from playwright.sync_api import Page
+from pytest_bdd import given, parsers, scenarios, then, when
+from .llm_studio_page import LLMStudioPage
+from .utils import login
+
+scenarios("llm_studio.feature")
+
+
+@given("LLM Studio home page is opened")
+def open_llm_studio(page: Page, base_url):
+    page.goto(base_url)
+
+
+@when("I login to LLM Studio", target_fixture="llm_studio")
+def login_to_llm_studio(logger: logging.Logger, page: Page, base_url: str):
+    okta_user = os.environ.get("OKTA_USER")
```

Comment:
what if we deploy LLM Studio to GCP env with keycloak instead of Okta?

Reply:
We need account for that yes. Do you know how to achieve this?

Reply:
jenkins has secrets for cloud-qa and managed cloud